src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/repository/impl/h2/DisputeRepositoryImpl.java

```diff
@@ -11,15 +11,32 @@
 import org.springframework.transaction.annotation.Transactional;
 
 import ca.bc.gov.open.jag.tco.oracledataapi.model.Dispute;
+import ca.bc.gov.open.jag.tco.oracledataapi.model.DisputeStatus;
 import ca.bc.gov.open.jag.tco.oracledataapi.repository.DisputeRepository;
 
 @ConditionalOnProperty(name = "ords.enabled", havingValue = "false", matchIfMissing = true)
 @Qualifier("disputeRepository")
-public interface DisputeRepositoryImpl extends DisputeRepository, JpaRepository<Dispute, Long> {
+public interface DisputeRepositoryImpl extends DisputeRepository, JpaRepository<Dispute, Long>, DisputeRepositoryCustom {
+
+	@Override
+	@Modifying
+	@Query("update Dispute d set d.userAssignedTo = :userName, d.userAssignedTs = CURRENT_TIMESTAMP() where d.disputeId = :disputeId")
+	@Transactional
+	public void assignDisputeToUser(@Param(value = "disputeId") Long disputeId, @Param(value = "userName") String userName);
```

Comment:
Moved assignDisputeToUser from DisputeService to this repo so it can get implemented differently for ORDS